rails_app/app/views/layouts/_header.html.haml

```diff
@@ -10,9 +10,22 @@
         = link_to 'Price monitor', root_url, class: 'navbar-brand'
```

Comment:
I guess we should add here our cool new logo:
<img width="426" alt="screen shot 2017-11-13 at 13 38 22" src="https://user-images.githubusercontent.com/7932772/32726342-f6128bfa-c878-11e7-8514-03297a1a774f.png">